fluent/test/primitives_test.js

```diff
@@ -143,41 +129,31 @@ suite('Primitives', function() {
     });
 
     test('can be used as a value', function(){
-      const msg = bundle.getMessage('bar');
-      const val = bundle.format(msg, args, errs);
+      const val = bundle.format('bar', args, errs);
       assert.equal(val, 'FooBar');
       assert.equal(errs.length, 0);
     });
 
-    test('is detected to be complex', function(){
-      const msg = bundle.getMessage('bar');
-      assert(Array.isArray(msg));
-    });
-
```

Comment:
It would probably be better to mark this test as skipped with `test.skip(...)` rather than just removing it.